webpack.dev.js

```diff
@@ -0,0 +1,15 @@
+const merge = require('webpack-merge');
+const config = require('./webpack.config.js');
+const WriteFilePlugin = require('write-file-webpack-plugin');
+
+module.exports = merge(config, {
+  mode: 'development',
+  devtool: 'source-map',
+  plugins: [
+    ...config.plugins,
+    new WriteFilePlugin()
+  ],
+  stats: {
+    colors: true
```

Comment:
🌈

Reply:
Yeah, I kept it from the original `webpack.js` file :smile: